packages/kbn-ecs/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@kbn/ecs",
+  "version": "1.0.0",
+  "private": true,
+  "author": "Kibana Core",
+  "license": "SSPL-1.0 OR Elastic License 2.0",
+  "main": "./target_node/index.js",
+  "types": "./target_types/index.d.ts"
```

Comment:
this is a stab in the dark, but im wondering if this should be `index.ts` instead of `index.d.ts`, since all the files youre feeding it are `.ts`